target-platform/target-platform.target

```diff
@@ -31,6 +31,7 @@
 			<unit id="ch.qos.logback.core" version="0.0.0"/>
 			<unit id="ch.qos.logback.classic" version="0.0.0"/>
 			<unit id="ch.qos.logback.slf4j" version="0.0.0"/>
+			<unit id="org.mockito" version="0.0.0"/>
```

Comment:
Mockito in Orbit has been maintained mainly by Platform needs but we moved to using the one from maven central directly. So this one is most likely stuck at this version and even worse uses different symbolic name compared to upstream.  It would be best to use approach similar to https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/blob/master/eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target#L322

Reply:
Thanks for that hint! I will change it.

Reply:
I think it would be better to just consume it from Platform, eg on line 11, than to re-consume it from Maven. This will give some consistency with Platform version.

Reply:
> I think it would be better to just consume it from Platform, eg on line 11, than to re-consume it from Maven. This will give some consistency with Platform version.

That's reasonable. I changed it accordingly.